sympy/vector/coordsys.py

```diff
@@ -85,6 +96,107 @@ def __new__(cls, name, location=None, rotation_matrix=None,
             location = Vector.zero
             origin = Point(name + '.origin')
 
+        default_vector_names = {
+            'cartesian' : ['i', 'j', 'k'],
+            'spherical' : ['e_r', 'e_theta', 'e_phi']
+        }
+
+        default_scalar_names = {
+            'cartesian' : ['x', 'y', 'z'],
+            'spherical' : ['r', 'theta', 'phi']
+        }
+
+        # Handling for different coordinate systems
+        coord_type_map = {
+            'cartesian': [
+                Lambda((x, y, z), (1, 1, 1)),
```

Comment:
I am to store coordinate relations of other coordinates with respect to `CartesianCoordinateSystem` so coordinate relation of `Cartesian System` should be an identity map.